### ISSUE TITLE
W191 False positive for multiline string

### ISSUE THREAD

**Issue by Nodd** [charliermarsh/ruff#3674]
Applying ruff on the following file (notice the tab indent in the string), ruff yields a error:
a.py :
```python
a="""
	a
"""
```
This is the command I executed:
```bash
$ ruff --version
ruff 0.0.257
$ ruff a.py --select W191
a.py:2:1: W191 Indentation contains tabs
Found 1 error.
```
Ruff should leave alone the text inside strings, since it could be anything.

**Comment by evanrittenhouse** [comment]
Duplicate of #3438 - I can pick this up

**Comment by Nodd** [comment]
Indeed, sorry for the duplicate!

**Comment by charliermarsh** [comment]
All good friend!
